tests/utils/mock_server.py

```diff
@@ -1849,6 +1849,12 @@ def history(self):
         history = fs_files.get("wandb-history.jsonl")
         return history
 
+    @property
+    def events(self):
```

Comment:
this doesn't seem to be used?

Reply:
ah yeah... i didn't end-up using it, just left it here, in case someone would need it. 
I tested it and it returns the correct values. Could remove if you think is better

Reply:
:+1: I'm fine with leaving it in, but I'm not sure what protocol is on unused testing utils.

Reply:
so i'll just remove it, probably safer

Reply:
Sounds good.